flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/KeyedStream.java

```diff
@@ -301,12 +301,12 @@ public KeyedStream(DataStream<T> dataStream, KeySelector<T, KEY> keySelector, Ty
 	}
 
 	/**
-	 * Applies an aggregation that that gives the current sum of the pojo data
+	 * Applies an aggregation that gives the current sum of the pojo data
```

Comment:
I would rewrite the entire description. This method does not only process "pojo data" but also tuples etc.